src/Voxel.MiddyNet/MiddyNetContext.cs

```diff
@@ -15,6 +16,7 @@ public class MiddyNetContext
         public Func<ILambdaLogger, IMiddyLogger> LoggerFactory { get; }
 
         public bool HasExceptions => HandlerException != null || MiddlewareBeforeExceptions.Count > 0 || MiddlewareAfterExceptions.Count > 0;
+        public TraceContext TraceContext { get; set; }
```

Comment:
Why do we need this? Can't we store the Tracecontext in the AdditionalContext like any other extra information the middlewares can add?

Reply:
The original idea was to obtain the new traceparent directly from the stored tracecontext via an instance method: 
`var newTraceContext = context.TraceContext.MutateParentId();`
It made sense to have the tracecontext stored as object instead of two properties ("traceparent" and "tracestate") in the AdditionalContext collection, a maybe 

The final implementation, though, uses an static method (in order to maintain coherence with other method like Reset).

In this scenario, the alternative you're proposing seems more clean, less intrusive, as the MiddyNetContext does not have to include a new object that can be null if no tracing is used.

I will review the solution to include this change.

Reply:
Yep, and in that way we´ll not force to include the NuGet package if the user doesn´t want it.

Reply:
Done. Now the traceparent, tracestate and traceid are stored in the AdditionalContext.